src/objects.js

```diff
@@ -1,41 +1,59 @@
 const createPerson = (name, age) => {
   // your code here
+  this.name = name;
+  this.age = age;
+  return this;
 };
 
 const getName = object => {
   // your code here
+  return object.name;
 };
 
 const getProperty = (property, object) => {
   // your code here
+  return object[property];
 };
 
 const hasProperty = (property, object) => {
   // your code here
+  return object[property]? true : false;
```

Comment:
Nice use of a ternary here! +1 You could also us the hasOwnProperty method https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwnProperty